manifests/params.pp

```diff
@@ -68,7 +68,7 @@
   }
 
   # Zabbix overall params. Is used by all components.
-  $zabbix_package_state                     = 'present'
+  $zabbix_package_state                     = 'latest'
```

Comment:
I'm not sure this is a good idea.  Nothing will automatically bump zabbix_version when updated packages get installed.